packages/astro-rss/README.md

```diff
@@ -233,6 +235,22 @@ export async function get(context) {
 }
 ```
 
+### `trailingSlash`
```

Comment:
Should this entire new section come around line 190, with the other configuration options for the `rss` function?

Reply:
Completely missed that, thank you